test/test_helper.rb

```diff
@@ -2,3 +2,4 @@
 SimpleCov.start
 require 'minitest/autorun'
 require 'minitest/pride'
+require 'mocha/minitest'
```

Comment:
Made the best use of stubs that I could.  The only one that was a little tricky was the very first stub for the random number assignment.